tests/introspection/tests.py

```diff
@@ -217,6 +217,21 @@ def test_get_constraints_indexes_orders(self):
                 indexes_verified += 1
         self.assertEqual(indexes_verified, 3)
 
+    @skipUnlessDBFeature('can_introspect_foreign_keys')
```

Comment:
This test should use ```@skipUnless(connection.vendor == 'sqlite', "This is an sqlite-specific issue")``` because it is specifically testing a regex in the sqlite backend's introspection.